packages/fast-components-react-base/src/select/select.props.ts

```diff
@@ -22,11 +17,13 @@ export interface SelectHandledProps extends SelectManagedClasses {
     /**
      * Custom function to render the trigger of the control
      * (ie. the ui that typically displays the current value and opens the menu in single select mode)
+     * Note: the optional triggerId param has been depracated and will always be null until it is removed
+     * in a future major version update
      */
     trigger?: (
         props: SelectProps,
         state: SelectState,
-        triggerId?: string
+        triggerId?: string //depracated
```

Comment:
nit: I think this is the first time we're deprecating a param, I found this from JSDoc as a solution for how to mark it as deprecated: https://github.com/jsdoc/jsdoc.github.io/issues/39

Reply:
Thanks, shows up in VS tooltips.